Request-Certificate.ps1

```diff
@@ -263,14 +264,23 @@ CertificateTemplate = "$TemplateName"
         write-verbose "Sending certificate request to CA"
         Write-Debug "CAName = $CAName"
             
-        if ($PSBoundParameters.ContainsKey('CAName')) {
-            Write-Debug "certreq -submit -config `"$CAName`" `"$req`" `"$cer`""
-            Invoke-Expression -Command "certreq -submit -config `"$CAName`" `"$req`" `"$cer`""
+        if (!$PSBoundParameters.ContainsKey('CAName')) {
+            $CAs = certutil -dump | Select-String 'Config:'
+            if (($CAs.Length -eq 1) -and ($CAs[0] -match '.*`(.*)''')) {
```

Comment:
The regex look a little strange to me and it actually not works for me (0 matches always) and so the script still ask to select a CA.

Reply:
Out of interest, what's the output of `certutil -dump`

Reply:
The out put is as follow:

```
PS C:\> certutil.exe -dump
Entry 0:
  Name:                         "Fellers Issuing CA"
  Organizational Unit:          ""
  Organization:                 "Fellers"
  Locality:                     ""
  State:                        ""
  Country/region:               "CH"
  Config:                       "CA01.intranet.fellers.ch\Fellers Issuing CA"
  Exchange Certificate:         ""
  Signature Certificate:        ""
  Description:                  ""
  Server:                       "CA01.intranet.fellers.ch"
  Authority:                    "Fellers Issuing CA"
  Sanitized Name:               "Fellers Issuing CA"
  Short Name:                   "Fellers Issuing CA"
  Sanitized Short Name:         "Fellers Issuing CA"
  Flags:                        "1"
  Web Enrollment Servers:       ""
CertUtil: -dump command completed successfully.
```

The Select-String parts seems also to work:
```
PS C:\> certutil -dump | Select-String 'Config:'

  Config:                       "CA01.intranet.fellers.ch\Fellers Issuing CA"
```